config/hsc/jointcal.py

```diff
@@ -7,3 +7,16 @@
 config.applyColorTerms = True
 
 config.colorterms.load(os.path.join(getPackageDir("obs_subaru"), "config", "hsc", "colorterms.py"))
+
+config.sourceSelector.name = 'science'
+config.sourceSelector['science'].doUnresolved = True
+config.sourceSelector['science'].doSignalToNoise = True
+config.sourceSelector['science'].signalToNoise.minimum = 10.
+config.sourceSelector['science'].doIsolated = False
```

Comment:
Why not cut to just isolated sources? I'm not sure I'd trust blended sources for either astrometry or photometry.

Reply:
Hmm. That's worth a rerun. I'll try that out.

Reply:
So the plan is to set the default for `doIsolated` to `True`?  I don't think I'd remove the option

Reply:
@RobertLuptonTheGood I don't follow which direction 
> I don't think I'd remove the option
is going.  You leaning True or False?